graphql/e2e/common/common.go

```diff
@@ -329,7 +331,7 @@ func containsRetryableCreateNamespaceError(resp *GraphQLResponse) bool {
 	return false
 }
 
-func CreateNamespace(t *testing.T, headers http.Header) uint64 {
+func CreateNamespace(t *testing.T, headers http.Header, customgraphAdminURLs ...string) uint64 {
```

Comment:
could this be custom*G*raphAdminURLs instead?

Reply:
Resolved